v8go.cc

```diff
@@ -1010,6 +1031,25 @@ RtnValue ObjectGet(ValuePtr ptr, const char* key) {
   return rtn;
 }
 
+RtnValue ObjectGetInternal(ValuePtr ptr, uint32_t idx) {
+  LOCAL_OBJECT(ptr);
+  RtnValue rtn = {nullptr, nullptr};
+
+  MaybeLocal<Value> result = obj->GetInternalField(idx);
```

Comment:
This returns a `Local<Value>`, so the `if (result.IsEmpty()) {` code path is dead code and we don't need to propagate an error for this call.